locale/areas.ru.tr

```diff
@@ -0,0 +1,135 @@
+# textdomain: areas
+
+
+
+### chatcommands.lua ###
+
+<AreaName>=<имя_территории>
+<ID> [faction_name]=<ID> [имя_фракции]
+<NewOwner>=<новый_владелец>
+<ParentID>=<ID_родительской>
+<PlayerName>=<имя_игрока>
+<newName>=<новое_имя>
+@1 has given you control over the area "@2" (ID @3).=@1 передал вам территорию "@2" (ID @3).
+@1 spanning up to @2x@3x@4.=@1 размером до @2x@3x@4.
+A regular expression is required.=Не указано регулярное выражение.
+Area @1 does not exist or is not owned by you.=Территория @1 не существует или Вы не имеете к ней доступа.
+Area closed for faction members.=Территория закрыта для фракций.
+Area closed.=Территория закрыта.
+Area does not exist.=Территория не существует.
+Area is open for members of: @1=Территория открыта для игроков из фракций: @1
+Area opened.=Территория открыта.
+Area protected. ID: @1=Территория защищена. ID: @1
+Area renamed.=Территория переименована.
+Area successfully moved.=Территория успешно перенесена.
+Change the owner of an area using its ID=Изменить владельца территории по её ID
+Faction doesn't exists=Фракция не существует.
+Find areas using a Lua regular expression=Поиск территорий с использованием регулярных выражений Lua
+Get information about area configuration and usage.=Получить информацию о конфигурации и использовании защиты территорий.
+
+Give a player access to a sub-area between two positions that have already been protected, Use set_owner if you don't want the parent to be set.=Дать игроку доступ к выделенной территории, создав дополнительный регион. Используйте set_owner, если вы не хотите привязки к родительской территории.
+
+Invalid regular expression.=Неверное регулярное выражение.
+Limit: @1 areas=Ограничение: @1 территорий(я)
+Limit: no area count limit=Ограничение: нет ограничений на кол-во территорий
+List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий (или всех существующих территорий, если Вы имеете привилегию areas)
```

Comment:
your translation  `List your available territories (or all existing territories if you have the areas privilege).` may try to match the `or all areas if you are an admin` part better